hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -166,6 +167,8 @@ public BlockOutputStream(
     this.xceiverClient = xceiverClientManager.acquireClient(pipeline);
     this.bufferPool = bufferPool;
     this.token = token;
+    this.tokenString = (this.token == null) ? null :
+        this.token.encodeToUrlString();
```

Comment:
The same improvement can be done in `BlockDataStreamOutput`:

https://github.com/apache/ozone/blob/9f475809d24598cd2b99fa62af93e638ab31860b/hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockDataStreamOutput.java#L210-L211

https://github.com/apache/ozone/blob/9f475809d24598cd2b99fa62af93e638ab31860b/hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockDataStreamOutput.java#L422-L423

https://github.com/apache/ozone/blob/9f475809d24598cd2b99fa62af93e638ab31860b/hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockDataStreamOutput.java#L439-L440